### PR TITLE
Prepare for 0.4.1 release

### PR DESCRIPTION
**Pull request by StephenWeatherford**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
 # Change Log
 All notable changes to the "vscode-azurearmtools" extension will be documented in this file.
 
+Version 0.4.1 (05/07/2018)
+
+- Add support for JSONC
```
Comment (PrashanthCorp) [inline CHANGELOG.md]: I would add a link to the spec/project here, and an example of extra capability it unlocks. 
Searching the web for JSONC, I see JSON+Comments, JSON with C-type reference counting, and JSON-Compressed. 
I personally expect the first, but a little confirmation would be nice. :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+- Improved labeling in JSON outline, similar to Visual Studio
+  - Use displayName tag for label of resources when available
+  - Shorten labels when possible (e.g. parameters['abc'] -> &lt;abc&gt;)
+- Engineering improvements (tests, lint, telemetry)
```
Comment (PrashanthCorp) [inline CHANGELOG.md]: Do I add this to the cosmosdb extension too? This would also apply there...

Comment (StephenWeatherford) [reply CHANGELOG.md]: A lot more happened in ARM.

File: CHANGELOG.md (hunk 1 above)
Comment (StephenWeatherford) [reply CHANGELOG.md]: Improved.

**Pull request merged**